modules/react/button/lib/ExternalHyperlink.tsx

```diff
@@ -12,26 +14,25 @@ export interface ExternalHyperlinkProps extends HyperlinkProps {
    */
   iconLabel?: string;
 }
+const relativeToFontSizeIconSize = '1em';
 
-const iconStyles = {
-  ...systemIconStyles({fill: 'currentColor', fillHover: 'currentColor'}),
-};
-
-const Anchor = styled(Hyperlink)<ExternalHyperlinkProps & StyledType>({
-  ...iconStyles,
-  display: 'inline-flex',
-  flexDirection: 'row',
-  alignItems: 'center',
-});
-
-const iconSize = '1em';
-const minIconSize = '16px';
-
-const StyledSystemIcon = styled(SystemIcon)<StyledType>({
-  ...iconStyles,
-  width: `calc(${iconSize} - 1px)`,
-  minWidth: `calc(${minIconSize} - 1px)`,
-  marginLeft: '2px',
+export const externalHyperlinkStencil = createStencil({
+  extends: hyperlinkStencil,
+  base: {
+    display: 'inline-flex',
+    flexDirection: 'row',
+    alignItems: 'center',
+    '& [data-part="external-hyperlink-icon"]': {
+      [systemIconStencil.vars.color]: 'currentColor',
+      width: calc.subtract(relativeToFontSizeIconSize, px2rem(1)),
+      minWidth: calc.subtract(system.space.x4, px2rem(1)),
+      marginInlineStart: calc.subtract(system.space.x1, px2rem(2)),
+      '& > svg': {
+        minWidth: system.space.x4,
+        minHeight: system.space.x4,
```

Comment:
I can't use `[systemIconStencil.vars.height]` because this is setting the `minWidth` and `minHeight`. This is kind of a unique instance, @NicholasBoll  any suggestions or is this good?

Reply:
I don't understand what you're asking. The `height` and `width` properties are set in SVG.tsx. Are you saying you really intend to set the `minHeight` and `minWidth` properties using system tokens because that is the intent?

The `size` is being set to `1em`. You're saying the `width` and `height` should be the same size as the font size, but should go no smaller than `x4`, right?

Reply:
> The size is being set to 1em. You're saying the width and height should be the same size as the font size, but should go no smaller than x4, right?

Correct